### DISCUSSION TITLE
Key Bindings

### DISCUSSION THREAD

**Discussion by owergame191** [seerge/g-helper#2754] [Q&A] [answered]
I can't open G-helper window on my laptop with binded button(FN+F4). When i use this bind my  mode is changes
, but this action was bound on another button. Reinstalling "Asus System Control Interface" does not help.
![image](https://github.com/seerge/g-helper/assets/122443693/08dfa394-dccd-467f-a465-82589a025e3a)

**Comment by seerge** [comment]
You need to stop Asus services, otherwise they handle hotkeys.

**Reply by owergame191** [reply]
Did it, but problem not solved.

**Reply by seerge** [reply, marked as answer]
Try to restart app after.

1. Do other keybindings work, could you open app window with default M5?
2. Does ``FN+F4`` work in general? I.e. does it cycle through backlight modes by default ?

You need to upload log from ``%appdata%\GHelper\log.txt`` to look further. ``FN+F4`` produces keycode 178 or 179, and it should be visible in the log.
